test/python/test_qi.py

```diff
@@ -25,7 +25,7 @@
 from qiskit.tools.qi.qi import partial_trace, vectorize, devectorize, outer
 from qiskit.tools.qi.qi import concurrence, qft, chop
 from qiskit.tools.qi.qi import shannon_entropy, entropy, mutual_information
-from qiskit.tools.qi.qi import choi_to_rauli
```

Comment:
This is an API change, right?

Reply:
this is a spelling fix.

Reply:
the fact that it was not caught, and even present in the corresponding unit test means that it is not widely used, so not worried.

Reply:
the fact we are not testing it is an issue by itself...

Users might have been using `choi_to_rauli`. I think we should deprecate it. But if you strongly disagree with that, I okey with that...